x-pack/plugins/ingest_manager/server/plugin.ts

```diff
@@ -231,6 +232,9 @@ export class IngestManagerPlugin
             );
           }
         } else {
+          // we currently only use this global interceptor if fleet is enabled
+          // since it would run this func on *every* req (other plugins, CSS, etc)
+          this.httpSetup.registerOnPreAuth(preAuthHandler);
```

Comment:
Should we include some other test to enable this global handler?

Reply:
`maxConcurrentConnections` now defaults to 0 (off) so this will require fleet enabled (default true) *and* `maxConcurrentConnections=somePositiveValue`